### ISSUE TITLE
Support for x64 device

### ISSUE THREAD

**Issue by thanasistrisp** [j-hc/mindetach-magisk#3]
I need support for x64 device as I have rooted Windows Subsystem for Android and this is the architecture it supports.

**Comment by j-hc** [comment]
what do you use mindetach for?

**Comment by Senliast** [comment]
Yes please, add support for x64, so it can be used on an emulator or WSA!
